node_helper.js

```diff
@@ -158,6 +169,26 @@ module.exports = NodeHelper.create({
         return dates.map(({date, games}) => games.map(game => ({...game, gameDay: date}))).flat();
     },
 
+    /**
+     * @function fetchPlayoffs
+     * @description Retrieves playoff data from the API.
+     * @async
+     *
+     * @returns {object} Raw playoff data from API endpoint.
+     */
+    async fetchPlayoffs() {
+        const response = await fetch(BASE_PLAYOFF_URL);
```

Comment:
```suggestion
        const response = await fetch(`${BASE_URL}/tournaments/playoffs?expand=round.series`);
```

Reply:
Feels better to keep this separate instead of assuming it will always be based on the same Base_Url